README.md

```diff
@@ -34,74 +34,39 @@ sed -i "s/02:FF:FF:FF:FF:FF/$mac_oui:FF:FF:FF/" kubemacpool.yaml
-- `mutatepods.kubemacpool.io=allocate` - to opt in pods mac allocation in your namespace
-- `mutatevirtualmachines.kubemacpool.io=allocate` - to opt in vms mac allocation in your namespace
+Kubemacpool is set to allocate mac to [supported interfaces](#About) on pods/vms that reside on all namespaces (opt-out). You can opt-out your namespace by adding the following labels:
+- `mutatepods.kubemacpool.io=ignore` - to opt-out pods mac allocation in your namespace
+- `mutatevirtualmachines.kubemacpool.io=ignore` - to opt out vms mac allocation in your namespace
 
-#### How to enable/disable kubemacpool for a namespace
+#### How to opt-out kubemacpool for a namespace
 
-Kubemacpool is disabled by default on a new namespace.
-To enable kubemacpool on a specific namespace:
+Kubemacpool is enabled by default on a new namespace.
+To disable kubemacpool on a specific namespace:
 ```bash
-kubectl label namespace example-namespace mutatepods.kubemacpool.io=allocate mutatevirtualmachines.kubemacpool.io=allocate
+kubectl label namespace example-namespace mutatepods.kubemacpool.io=ignore mutatevirtualmachines.kubemacpool.io=ignore
 namespace/example-namespace labeled
 ```
 
-To disable kubemacpool in a namespace:
+To re-enable kubemacpool in a namespace:
 ```bash
 kubectl label namespace example-namespace mutatepods.kubemacpool.io- mutatevirtualmachines.kubemacpool.io-
 namespace/example-namespace labeled
 ```
 
 **note:** If a VMI is created directly and not through a VM, then it is handled in kubemacpool by the pod handler.
 
-#### How to change the opt-in label value on kubemacpool
-
-The kubemacpool opt-in label and value is set in the mutatingwebhookconfiguration instance called `kubemacpool-mutator` by a `namespaceSelector`.
-In order to change the opt-in label value - you need to edit the mutatingwebhookconfiguration instance and change the label value accordingly. This can be done separately for vms and pods.
-```yaml
-apiVersion: admissionregistration.k8s.io/v1
-kind: MutatingWebhookConfiguration
-metadata:
- ...
-  name: kubemacpool-mutator
- ...
-webhooks:
-- admissionReviewVersions:
- ...
-  name: mutatepods.kubemacpool.io
-  namespaceSelector:
- ...
-    matchLabels:
-      mutatepods.kubemacpool.io: allocate
-- admissionReviewVersions:
- ...
-  name: mutatevirtualmachines.kubemacpool.io
-  namespaceSelector:
- ...
-    matchLabels:
-      mutatevirtualmachines.kubemacpool.io: allocate
- ...
-```
-
-**note:** if the kubemacpool's mutatingwebhookconfiguration `kubemacpool-mutator` namespace-selector value per vm/pod is set to `allocate`, then you can also opt-out your namespace by setting the label value to `disable` in your namespace:
-```bash
-kubectl label namespace example-namespace --overwrite mutatepods.kubemacpool.io=disable mutatevirtualmachines.kubemacpool.io=disable
-namespace/example-namespace labeled
-```
-
-#### Opt-in Example
+#### Opt-out Example
 
 ```bash
 # Add the opt-in labels to namespace using kubectl
```

Comment:
opt-out

Reply:
Do you mean remove the Capital 'O'?

Reply:
Done

Reply:
> The YAML duplication is terrible, but we don't need to tackle it here. Thanks!

I know, I tried using symbolic link but kustomize didn't eat it.